lib/awful/hotkeys_popup/keys/firefox.lua

```diff
@@ -19,7 +19,7 @@ local firefox_keys = {
     ["Firefox: tabs"] = {{
         modifiers = { "Mod1" },
         keys = {
-            ["1..9"] = "go to tab"
+            ["1…9"] = "go to tab"
```

Comment:
I think these changes should be in their own commit and own pull request. Some of what is proposed here might drag on since it is going to be rather controversial.